Source/bmalloc/bmalloc/DebugHeap.cpp

```diff
@@ -122,7 +122,11 @@ void* DebugHeap::malloc(size_t size, FailureAction action)
 
 void* DebugHeap::memalign(size_t alignment, size_t size, FailureAction action)
 {
+    #if !BOS(WINDOWS)
     void* result = ::aligned_alloc(alignment, size);
+    #else
+    void* result = ::_aligned_malloc(alignment, size);
```

Comment:
Note that it does not work: the memory needs to be deallocated via _aligned_free in Windows and we cannot use free. So it is not possible to implement the same interface with Windows default library. So on Windows, probably we should disable system malloc fallback mechanism